nestkernel/connector_model_impl.h

```diff
@@ -463,6 +476,19 @@ GenericConnectorModel< ConnectionT >::add_connection( Node& src,
     }
   }
 
+  // Let connections without delay contribute to the delay extrema with
+  // wfr_comm_interval. For those connections the min_delay is important
+  // as it determines the length of the global communication interval.
+  // The call to assert_valid_delay_ms needs to happen only once
+  // (either here or in used_default_delay()) when the first connection
+  // without delay is created.
+  if ( default_delay_needs_check_ && not has_delay_ )
+  {
+    kernel().connection_manager.get_delay_checker().assert_valid_delay_ms(
+      kernel().simulation_manager.get_wfr_comm_interval() );
+    default_delay_needs_check_ = false;
+  }
+
```

Comment:
This solution looks very good. Maybe it should be placed at the start of the function (line 354ff), because if something is bad with the delay, the connection is not stored in the connection infrastructure.